novuhq/website
# feat: add inbox component

**Describe what changes this pull request brings**

Add new interactive inbox component

**Steps to check:**

1. Open preview: [test page](https://deploy-preview-259--novu-website.netlify.app/testpages/reusable-components/), [inbox page](https://deploy-preview-259--novu-website.netlify.app/inbox/)
2. Make sure that everything looks and works as expected
